apps/src/p5lab/AnimationTab/PiskelEditor.jsx

```diff
@@ -61,6 +62,9 @@ class PiskelEditor extends React.Component {
     this.piskel.onPiskelReady(this.onPiskelReady);
     this.piskel.onStateSaved(this.onAnimationSaved);
     this.piskel.onAddFrame(this.onAddFrame);
+
+    const iframe = document.getElementById('piskelFrame');
+    iframe.contentWindow.locale = getStore().getState().locales.localeCode;
```

Comment:
I thought the place for `getStore().getState().locales...` was supposed to be in the `connect` method below.

And then this code would be `iframe.contentWindow.locale = this.props.locale`